From 804c50536e1e6f87f19879f7350fc36f2035e414 Mon Sep 17 00:00:00 2001
From: Frank Chen <FrankChen4@quantatw.com>
Date: Mon, 27 Jul 2026 08:52:54 +0800
Subject: [PATCH] configurations: catalina_fio: update FIO thresholds

Santa Barbara and Catalina use the same FIO board but have
different thermal threshold requirements.

Update the Catalina FIO threshold configuration with the values
provided by the Santa Barbara thermal team.

Gerrit: https://gerrit.openbmc.org/c/openbmc/entity-manager/+/93696

Change-Id: I695a9cfd2aa807ede09b3233077b28286c50ae24
Signed-off-by: Frank Chen <FrankChen4@quantatw.com>
---
 .../meta/catalina/catalina_fio.json           | 30 ++++++++++++++-----
 1 file changed, 22 insertions(+), 8 deletions(-)

diff --git a/configurations/meta/catalina/catalina_fio.json b/configurations/meta/catalina/catalina_fio.json
index a5150bf..0925d01 100644
--- a/configurations/meta/catalina/catalina_fio.json
+++ b/configurations/meta/catalina/catalina_fio.json
@@ -13,17 +13,24 @@
             "Thresholds": [
                 {
                     "Direction": "greater than",
-                    "Hysteresis": 0.9,
+                    "Hysteresis": 0.84,
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 45
+                    "Value": 42
+                },
+                {
+                    "Direction": "greater than",
+                    "Hysteresis": 0.74,
+                    "Name": "upper non critical",
+                    "Severity": 0,
+                    "Value": 37.0
                 },
                 {
                     "Direction": "less than",
-                    "Hysteresis": 0.1,
+                    "Hysteresis": 0.04,
                     "Name": "lower critical",
                     "Severity": 1,
-                    "Value": 5
+                    "Value": -2
                 }
             ],
             "Type": "TMP75"
@@ -35,17 +42,24 @@
             "Thresholds": [
                 {
                     "Direction": "greater than",
-                    "Hysteresis": 0.9,
+                    "Hysteresis": 0.84,
                     "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 45
+                    "Value": 42
+                },
+                {
+                    "Direction": "greater than",
+                    "Hysteresis": 0.74,
+                    "Name": "upper non critical",
+                    "Severity": 0,
+                    "Value": 37.0
                 },
                 {
                     "Direction": "less than",
-                    "Hysteresis": 0.1,
+                    "Hysteresis": 0.04,
                     "Name": "lower critical",
                     "Severity": 1,
-                    "Value": 5
+                    "Value": -2
                 }
             ],
             "Type": "TMP75"
-- 
2.43.0

